### DISCUSSION TITLE
Jet Brains Rider Gauge plugin

### DISCUSSION THREAD

**Discussion by zdmta** [getgauge/gauge#2462] [General]
Are there plans to have Rider plug in for. Gauge?
How difficult is to create such plug-in? Thanks

**Comment by sriv** [comment]
There are no plans to build support for Rider. There is an Intellij Plugin for gauge which is now maintained by Jetbrains - https://github.com/JetBrains/intellij-plugins/tree/master/gauge

You may check if there is any plan in their roadmap for this.

**Reply by zdmta** [reply]
hi sirv, many thanks for your answer. I really like Gauge and we are using Rider for development, thats why I asked,
